src/components/App.tsx

```diff
@@ -15,7 +15,11 @@ export default class App extends React.Component<{}, {}> {
 
 		initializeLocations();
```

Comment:
Should locations be added to Redux?

Idk. Definitely not as part of this PR but... should they?

Gosh.